src/Session.cc

```diff
@@ -1387,10 +1390,23 @@ bool Session::SendSpatialProfileData(int file_id, int region_id) {
                 data_sent = true;
             }
         }
+    } else {
+        string error = fmt::format("Spatial profiles not valid for non-cursor or non-point region ", region_id);
```

Comment:
Minor text correction: `non-cursor or non-point` -> `non-cursor and non-point`

Or maybe just `non-point`, since a cursor region is a kind of point region?

Reply:
right, `non-point` should be sufficient.